### PR TITLE
PICARD-1365: Working Windows builds with PyQt 5.11

### PR DESCRIPTION
**Pull request by phw**
# Summary

Fixes building with PyQt 5.11 on Windows. The core change is to use pyinstaller 3.4, which has PyQt related fixes. Also uses the openssl DLLs bundled with PyQt, since we can assume those are compatible with that specific PyQt release.

This needs testing outside my own dev environment, opening a PR to trigger an Appveyor build.

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1365](https://tickets.metabrainz.org/browse/PICARD-1365)

# Solution

# Action

### CODE REVIEW

**Comment by phw** [comment]
@rdswift I would appreciate your feedback and testing of the builds at https://ci.appveyor.com/project/MetaBrainz/picard/build/2.0.0.dev.2.249-edbvrmcl

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 discid
 mutagen>=1.37
-PyQt5>=5.7.1,<5.11
+PyQt5>=5.7.1
```
Comment (samj1912) [inline requirements.txt]: I would actually like to freeze the build dependencies. This prevents unknown regressions (this happened during the 2.0.1 release with osx). Maybe a requirements-prod-win or something like that? And similar file for osx.

**Comment by phw** [comment]
Travis is currently misbehaving and hangs on apt install, but not only for this branch :(

**Comment by rdswift** [comment]
Sorry for not looking at this until now.  I've been in meetings all day and just got home.  I'll look at it right away.

**Comment by samj1912** [comment]
I think we can move the prod freeze to another PR. This does too many other things as it is (changing ssl binaries, pyinstaller version, qt5 version).

**Comment by phw** [comment]
@samj1912 Yep, I have taken that commit out of this PR. I will submit a separate PR once this one has been merged.

**Comment by phw** [comment]
I am going to merge the windows changes and ask on the forums for broader testing

**Pull request merged**